### PR TITLE
Selected content as article body

### PR DESCRIPTION
**Pull request by greatghoul**
Sometimes the readability parser can not grab the correct content of an article,
users can selected what they want to read, then read it in Reader View

### CODE REVIEW

File: WebExtension/data/inject/wrapper.js [hunk 1]
```diff
@@ -1,7 +1,45 @@
 /* globals Readability */
 'use strict';
 
-var article = new Readability(document.cloneNode(true)).parse();
+// The implementation is from https://stackoverflow.com/a/5084441/260793
+function getSelectionHTML() {
+  var userSelection;
+  if (window.getSelection) {
```
Comment (rNeomy) [inline WebExtension/data/inject/wrapper.js]: There is no need for this. All the supported browsers do support `window.getSelection()`

Comment (greatghoul) [reply WebExtension/data/inject/wrapper.js]: okay, thank you, I will change and push later. :]

**Comment by greatghoul** [comment]
@rNeomy  I have updated.

**Comment by rNeomy** [comment]
It looks really good! Please review my [minor changes](https://github.com/rNeomy/reader-view/commit/8657181a785e91fad529e470cbecb788713a937b) and if you are happy with them I'll merge!

1. remove `innerHTML` usage
2. return `doc` element only if the selection is not empty
3. return the entire paragraph instead (`range.commonAncestorContainer`). So if a user selects just a small portion, still we display the right content

**Comment by greatghoul** [comment]
it looks good to me. 👍

**Comment by rNeomy** [comment]
Perfect! It is merged

**Pull request merged**